aspoet/src/main/kotlin/com/google/androidstudiopoet/generators/android_modules/AndroidModuleGenerator.kt

```diff
@@ -43,6 +44,10 @@ class AndroidModuleGenerator(private val resourcesGenerator: ResourcesGenerator,
         packagesGenerator.writePackages(blueprint.packagesBlueprint)
         blueprint.activityBlueprints.forEach({ activityGenerator.generate(it) })
         manifestGenerator.generate(blueprint)
+
+        if (blueprint.generateBazelFiles != null && blueprint.generateBazelFiles) {
```

Comment:
TBH my idea was bit different. I thought that if `blueprint.buildBazelBlueprint` is optional then this code can look something like `blueprint.buildBazelBlueprint?.let { buildBazelGenerator.generate(it) }, then generator doesn't need to know about `blueprint.generateBazelFiles`

Reply:
https://github.com/android/android-studio-poet/pull/134